File: demo/src/main/java/com/blunderer/easyanimatedvectordrawabledemo/MainActivity.java
package com.blunderer.easyanimatedvectordrawabledemo;

import android.graphics.Color;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.AppCompatImageView;
import android.view.View;
import android.widget.Button;

import com.blunderer.easyanimatedvectordrawable.EasyAnimatedVectorDrawable;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button playButton = findViewById(R.id.activity_main_play_button);
        Button pauseButton = findViewById(R.id.activity_main_pause_button);
        Button stopButton = findViewById(R.id.activity_main_stop_button);

        final AppCompatImageView imageView = findViewById(R.id.activity_main_imageview);

        playButton.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                EasyAnimatedVectorDrawable.setImageType(imageView, EasyAnimatedVectorDrawable.Type.PLAY, Color.RED);
            }

        });
        pauseButton.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                EasyAnimatedVectorDrawable.setImageType(imageView, EasyAnimatedVectorDrawable.Type.PAUSE, Color.RED);
            }

        });
        stopButton.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                EasyAnimatedVectorDrawable.setImageType(imageView, EasyAnimatedVectorDrawable.Type.STOP, Color.RED);
            }

        });
    }

}

File: demo/src/main/java/com/blunderer/easyanimatedvectordrawabledemo/MainActivity.kt
package com.blunderer.easyanimatedvectordrawabledemo

import android.graphics.Color
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.blunderer.easyanimatedvectordrawable.EasyAnimatedVectorDrawable
import com.blunderer.easyanimatedvectordrawable.setImageType
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        activity_main_play_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.PLAY, Color.RED) }
        activity_main_pause_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.PAUSE, Color.RED) }
        activity_main_stop_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.STOP, Color.RED) }

        activity_main_left_arrow_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.LEFT_ARROW, Color.BLUE) }
        activity_main_up_arrow_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.UP_ARROW, Color.RED) }
        activity_main_right_arrow_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.RIGHT_ARROW) }
        activity_main_down_arrow_button.setOnClickListener { activity_main_imageview.setImageType(EasyAnimatedVectorDrawable.Type.DOWN_ARROW, Color.RED) }
    }

}
